ledger/src/blockstore.rs

```diff
@@ -7618,7 +7617,12 @@ pub mod tests {
         let slot = 1;
         let (_data_shreds, code_shreds, _) =
             setup_erasure_shreds_with_index_and_chained_merkle_and_last_in_slot(
-                slot, 0, 10, 0, None, true,
+                slot,
+                0,
+                10,
+                0,
+                Hash::default(),
```

Comment:
note to other reviewers - there are other instances where we use `Hash::default()` so changing `None` to that here shouldn't be an issue.

No code changes being requested, just calling this out